# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rocsolver/hotfix-libdir.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rocSOLVER-rocm-6.4.0/library/src/CMakeLists.txt.vanilla	2025-04-12 19:15:28.633493215 +0200
+++ rocSOLVER-rocm-6.4.0/library/src/CMakeLists.txt	2025-04-12 19:17:11.391197950 +0200
@@ -393,6 +393,9 @@
   target_link_libraries(rocsolver PRIVATE fmt::fmt)
 endif()
 
+target_link_libraries(rocsolver PRIVATE "-L/opt/rocm/lib64 -L/opt/rocm/lib")
+
+
 # In ROCm 4.0 and earlier, the default maximum threads per block is 256
 target_compile_options(rocsolver PRIVATE --gpu-max-threads-per-block=1024)
 
